### PR TITLE
backup current checks

### PR DESCRIPTION
**Pull request by w8896699**
looks like its been a while since last backup

### CODE REVIEW

**Comment by ShellyXueHan** [comment]
pls get rid of the monitors that are part of our teams'.

**Comment by w8896699** [comment]
@ShellyXueHan I have applied our team filter on search results this time. Thanks for reminding me.

File: monitor-setup/checks.json [hunk 1]
```diff
@@ -1,643 +1,656 @@
...
+      "msp_expect_string": "",
+      "msp_expect_string_type": "STRING",
+      "msp_encryption": "",
+      "msp_threshold": 60,
+      "msp_headers": "",
+      "msp_script": "[{\"step_def\":\"C_OPEN_URL\",\"values\":{\"url\":\"https://registry.developer.gov.bc.ca/\"}},{\"step_def\":\"C_WAIT_FOR_ELEMENT_TEXT\",\"values\":{\"element\":\"button\",\"text\":\"Login\",\"is_regex\":\"\"}},{\"step_def\":\"C_MOUSE_CLICK\",\"values\":{\"element\":\".css-1jhnpo2\",\"button\":\"left\",\"click_count\":\"1\",\"skip_navigation\":\"\"}},{\"step_def\":\"C_WAIT_FOR_ELEMENT\",\"values\":{\"element\":\"#password\"}},{\"step_def\":\"C_FILL_FIELD\",\"values\":{\"element\":\"#user\",\"text\":\"psta\",\"typing_delay\":\"\"}},{\"step_def\":\"C_FILL_FIELD\",\"values\":{\"element\":\"#password\",\"text\":\"p1aTf0rmS3rV!c3sT3Am\",\"typing_delay\":\"\"}},{\"step_def\":\"C_MOUSE_CLICK\",\"values\":{\"element\":\".login-form-action > .btn-primary\",\"button\":\"left\",\"click_count\":\"1\",\"skip_navigation\":\"\"}},{\"step_def\":\"C_WAIT_FOR_ELEMENT_TEXT\",\"values\":{\"element\":\".css-lgt0d3 .css-vurnku > .css-vurnku\",\"text\":\"Git documentation for Platform Services Registry\",\"is_regex\":\"\"}}]",
```
Comment (ShellyXueHan) [inline monitor-setup/checks.json:642]: you'll need to remove the credential for the IDIR account! Either revert commits and get rid of the history here, or you might have to update the test IDIR account credential.

**Review by ShellyXueHan — CHANGES_REQUESTED** [review]

**Comment by w8896699** [comment]
I delete last comment as well because credential can still be seen there. @ShellyXueHan

**Review by ShellyXueHan — APPROVED** [review]

**Pull request merged**